testdata/p4_14_errors_outputs/nasty_meta.p4-stderr

```diff
@@ -0,0 +1,4 @@
+nasty_meta.p4(6): [--Werror=invalid] error: struct ethernet_t: Invalid expected a header
```

Comment:
Sorry to be such a nuisance, but I think that the error should point to the extract and not to the header type declaration.

Reply:
No nuisance, I actually appreciate it! Better error messages are very important.